### ISSUE TITLE
About the samples_per_gpu and learning rate of Cityscapes dataset

### ISSUE THREAD

**Issue by AGanPoi** [zhanggang001/RefineMask#34]
I see the _samples_per_gpu_ of Cityscapes is change to 2, but the learning rate is still set to 0.01. I understand the modification of _samples_per_gpu_ of COCO dataset, for the batchsize is 16 (8 GPU * 2 _samples_per_gpu_) and corresponding to learning rate of 0.02. But I wonder when the _samples_per_gpu_ of Cityscapes dataset is 2, the learning rate should also change?

**Comment by zhanggang001** [comment]
The total batch size for training Cityscapes is 8 by default. So if the samples_per_gpu is set to 2, the number of GPUs should be set to 4.

**Comment by AGanPoi** [comment]
已收到 甘伟泽
